lib/airbrake/logger/logger_ext.rb

```diff
@@ -14,30 +14,30 @@
 class Logger
   # Store the orginal methods to use them later.
   alias add_without_airbrake add
-  alias initialize_without_airbrake initialize
-
-  ##
-  # @see https://goo.gl/MvlYq3 Logger#initialize
-  def initialize(*args)
-    @airbrake_notifier = Airbrake[:default]
-    @airbrake_severity_level = WARN
-    initialize_without_airbrake(*args)
-  end
 
   ##
   # @return [Airbrake::Notifier] notifier to be used to send notices
   attr_accessor :airbrake_notifier
 
   ##
+  # @return [Integer]
+  attr_reader :airbrake_severity_level
+
+  ##
+  # Sets airbrake severity level. Does not permit values below `Logger::WARN`.
+  #
   # @example
   #   logger.airbrake_severity_level = Logger::FATAL
-  # @return [Integer]
-  attr_accessor :airbrake_severity_level
+  # @return [void]
+  def airbrake_severity_level=(level)
+    level = WARN if level < WARN
+    @airbrake_severity_level = level
+  end
 
   ##
   # @see https://goo.gl/8zPyoM Logger#add
   def add(severity, message = nil, progname = nil, &block)
-    if severity >= airbrake_severity_level && airbrake_notifier
+    if severity >= current_airbrake_severity && current_airbrake_notifier
       notify_airbrake(severity, message || progname)
```

Comment:
It is confusing that you check for `current_airbrake_notifier`, but don't use it.